content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -14,22 +14,31 @@ Datadog uses a graph processing framework to map relationships between cloud res
 
 For more information on network reachability, see the [AWS documentation][34] and the [AWS Network Reachability Analyser][35]. Currently, the `Is Publicly Accessible` facet is only available for AWS resources.
 
+This page details on a resource-by-resource basis the logic behind public accessibility determination for a resource.
```

Comment:
```suggestion
```

Reply:
I would remove this line as it's a duplicate of the second sentence in the initial paragraph.